config/app.config

```diff
@@ -28,6 +28,8 @@ machine_type:
     bcl2fastq_version: 2.17.1
   MiSeq:
     bcl2fastq_version: 2.17.1
+  NovaSeq:
+    bcl2fastq_version: 2.17.1
```

Comment:
is this just an example config? otherwise, I believe we're on `2.20.x`

Reply:
Yes. This is just an example config. One can specify the paths to multiple versions in the config as well, so I'm not sure we need to keep this particular one up to date.